src/python/pants/backend/python/subsystems/repos.py

```diff
@@ -60,6 +63,42 @@ class PythonRepos(Subsystem):
         advanced=True,
     )
 
+    path_mappings = StrListOption(
+        help=softwrap(
+            f"""
+            Mappings to facilitate using local Python requirements when the absolute file paths
+            are different on different users' machines. For example, the
+            path `file:///Users/pantsbuild/prebuilt_wheels/django-3.1.1-py3-none-any.whl` could
+            become `file://${{WHEELS_DIR}}/django-3.1.1-py3-none-any.whl`, where each user can
+            configure what WHEELS_DIR points to on their machine.
+
+            Expects values in the form `NAME|PATH`, e.g.
+            `WHEELS_DIR|/Users/pantsbuild/prebuilt_wheels`. You can specify multiple
+            entries in the list.
+
+            This feature is intended to be used with `[python-repos].find_links`, rather than PEP
+            440 direct reference requirements (see
+            {doc_url("python-third-party-dependencies#local-requirements")}.
+            `[python-repos].find_links` must be configured to a valid absolute path for the
+            current machine.
+
+            Tip: you can avoid each user needing to manually configure this option and
```

Comment:
This is more info than we need to put here, since we link to the docs anyway. Let's not cognitively overload the user here.